misk/web/tabs/web-actions/src/ducks/webActions.ts

```diff
@@ -197,10 +209,11 @@ function* handleMetadata() {
           authFunctionAnnotations,
           dispatchMechanism: [action.dispatchMechanism],
           function: action.function.split("fun ").pop(),
+          hash: groupByWebActionHash(action),
           nonAccessOrTypeFunctionAnnotations
         }
       })
-      .groupBy(groupByWebActionHash)
+      .groupBy("hash")
```

Comment:
this is pretty magical, is there something more deterministic to use here?

Reply:
The `groupByWebActionHash` function previously referenced now saves the value in a key in the object called `"hash"`. Maybe a different named key would seem less magical.

Reply:
we're relying on the fact that the key is "hash". Can we name it something more relevant and extract that to not a magic string?